Bonusopdracht.js

```diff
@@ -0,0 +1,83 @@
+// Bonusopdracht (1.)
+
+//INPUT
+// -
+
+//OUTPUT
+// Drie aangemaakte knoppen:
+// Sorteer op prijs
+// AmbiLight TV's
+// Uitverkochte exemplaren
+
+//Resulten worden in console gelogt
+
+//STAPPEN
+// Drie buttons maken in HTML
+// Drie functies schrijven a.d.h.v. code uit opdracht 1
+// Knoppen koppelen aan de functies in HTML
+
+
+function sortTelevisionsOnPriceInConsole() {
+    inventory.sort((a, b) => {
+        return a.price - b.price;
+    })
+    return inventory.map((item) => {
+        console.log("TV naam: " + item.name + " - Prijs: " + item.price);
+    })
+}
+
+function getAmbiLightTelevisionsInConsole() {
+    return inventory.filter((television) => {
+        if (television.options.ambiLight) {
+            console.log("AmbiLight TV's: " + television.name);
+        }
+    })
```

Comment:
De filter functie verwacht dat er een boolean teruggegeven wordt. Bij `true` geeft hij uiteindelijk die televisie wél terug, en bij `false` niet:
```suggestion
    return inventory.filter(television => television.options.ambiLight);
```